explorer/src/Stories/Tooltip.elm

```diff
@@ -40,6 +40,30 @@ stories =
                     ]
           , {}
           )
+        , ( "Top with arrow placed near corner"
+          , \_ ->
+                div []
+                    [ text "There is a tooltip "
+                    , Tooltip.init
+                        |> Tooltip.withPlacement Top
+                        |> Tooltip.withArrowPlacement NearCorner
+                        |> Tooltip.withContent (Tooltip.infoTooltip [] [ text "This is a tooltip" ])
+                        |> Tooltip.view [] [ text "here" ]
+                    ]
+          , {}
+          )
+        , ( "Top with arrow placed near other corner"
+          , \_ ->
+                div []
+                    [ text "There is a tooltip "
+                    , Tooltip.init
+                        |> Tooltip.withPlacement Top
+                        |> Tooltip.withArrowPlacement NearOtherCorner
```

Comment:
Instead of introducing another field, what do you think about extending the existing? One benefit is that we get more clear naming, instead of `Top` and `NearCorner`, `Center`, `NearOtherCorner`, we can for example have `TopLeft`, `Top` (or TopCenter), and `TopRight`. (And for example `RightBottom` for bottom corner on the right side)

What do you think?
```suggestion
                        |> Tooltip.withPlacement TopLeft
```

Reply:
Css already have similar naming scheme like this suggestion for corners: see https://package.elm-lang.org/packages/rtfeldman/elm-css/latest/Css#borderBottomRightRadius